index.bs

```diff
@@ -10574,14 +10574,20 @@ Operations that happen <dfn>atomically</dfn> on a
 given thread can only be executed when no other [=Atomically|atomic=]
 operation is running on another thread.
 
-The algorithm for rendering a block of audio from an
-{{AudioContext}} <em>G</em> with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is as
-follows.
-
-If the algorithm returns true then it MUST be executed again in the
-future, to render the next block of audio. Else, the <a>rendering
-thread</a> yields and the processing stops. The <a>control thread</a>
-can restart executing this algoritm if needed.
+The algorithm for rendering a block of audio from an {{AudioContext}} <em>G</em>
+with a <a href="#control-message-queue">control message queue</a> <em>Q</em> is
+comprised of multiple steps and the order of these steps cannot be altered or
+interrupted:
```

Comment:
I think I'd just says "comprised of multiple steps as follows and explained in further detail below in <link to algorithm.>:"  These steps really lack detail but you fill them in in the algorithm below, so linking would be great.

Reply:
Done.